controller/workitem_blackbox_test.go

```diff
@@ -801,11 +801,10 @@ func createOneRandomIteration(ctx context.Context, db *gorm.DB) *iteration.Itera
 		return nil
 	}
 
-	userActive := false
 	itr := iteration.Iteration{
 		Name:       "Sprint 101",
 		SpaceID:    space.ID,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c